GoogleForCreators/web-stories-wp
# Story Editor: Update Font Picker to Designs

## Summary
Updates the font picker to reflect the style changes made in [these figma designs](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=937%3A0)

Also updates other drop downs to reflect style changes in font picker described in [this ticket](https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/4059)

## To-do
~Follow up on expected behavior with designs~

## User-facing changes
- Font selector has updated styling.
- Media drop down, font weight drop down & author drop down in `document` tab reflect the same changes in the font selector

## Testing Instructions
Open up any editor and alter/change the font family. See that it's styled like [these designs](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=710%3A0) & works as it did before.

Also check:
- Media drop down
- Font weight drop down
- Author drop down (in document tab)

And see that they visually resemble the font picker and still function as expected

## Screenshots
<img width="328" alt="Screen Shot 2020-09-01 at 2 54 58 PM" src="https://user-images.githubusercontent.com/35983235/91905081-339bfe00-ec63-11ea-9316-22a56ecf6472.png">
<img width="370" alt="Screen Shot 2020-09-01 at 2 55 10 PM" src="https://user-images.githubusercontent.com/35983235/91905088-34cd2b00-ec63-11ea-8baf-43bde9b8ad6d.png">
<img width="321" alt="Screen Shot 2020-09-01 at 2 55 23 PM" src="https://user-images.githubusercontent.com/35983235/91905090-3696ee80-ec63-11ea-8643-b09ffdc70aea.png">
<img width="189" alt="Screen Shot 2020-09-02 at 2 14 45 PM" src="https://user-images.githubusercontent.com/35983235/92032402-51329b80-ed27-11ea-8713-7f7f9729bd03.png">





Fixes #4009 
(also fixes #4059  )
